### PR TITLE
feat: get jwk from keeper

### PR DESCRIPTION
**Pull request by shiki-tak**
## Description

- Save jwks in the keeper field.
- If you want to use jwk for verify, please use GetJWK.
- If there is no corresponding kid when verifying, please run FetchJWK.

## Checklist:

- [ ] I followed the [contributing guidelines](https://github.com/Finschia/finschia-sdk/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/Finschia/finschia-sdk/blob/main/CODE_OF_CONDUCT.md).
- [ ] I have added a relevant changelog to `CHANGELOG.md`
- [ ] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated API documentation `client/docs/swagger-ui/swagger.yaml`

### CODE REVIEW

File: x/zkauth/keeper/keeper.go [hunk 1]
```diff
@@ -105,18 +90,61 @@ func (k Keeper) ParseJWKs(byteArray []byte) (jwks []types.JWK, err error) {
...
+	if err != nil {
+		return err
+	}
+
+	_, err = io.Copy(file, resp.Body)
+	if err != nil {
+		return err
+	}
+
+	dataBytes, err := os.ReadFile(targetFile)
```
Comment (Mdaiki0730) [inline x/zkauth/keeper/keeper.go:115]: How about parsing the resp as is?

**Review by zemyblue — COMMENTED** [review]
I think we need to do refactoring to fetch class in keeper. 
Because it is a little difficult to fetch jwk in antehandler.

File: x/zkauth/keeper/keeper.go (hunk 1 above)
Comment (shiki-tak) [reply x/zkauth/keeper/keeper.go]: I changed the order of assigning to dataBytes and use WriteFile.

Comment (Mdaiki0730) [reply x/zkauth/keeper/keeper.go]: The key is held in both the jwks field and the file, so I don't think it is necessary to save it to the file. What do you think?
https://github.com/Finschia/finschia-sdk/pull/1249/files#diff-2a2241d04eae2d37982aafdf8f50d27ae141f623782c48ee72fbe125ece11fc7R23

Comment (shiki-tak) [reply x/zkauth/keeper/keeper.go]: I thought about it again, and it doesn't seem necessary to be stored in both as you say. I stopped saving locally.

**Pull request closed without merging**